app/proto/search.proto

```diff
@@ -101,6 +101,18 @@ message SearchRes {
   string next_page_token = 2;
 }
 
+enum LanguageFluency {
+    BEGINNER = 0;
+    CONVERSATIONAL = 1;
+    FLUENT = 2;
+}
```

Comment:
Please import `api.proto`, then you should be able to use `org.couchers.api.core.LanguageAbility.Fluency` I believe.